core/iwasm/common/wasm_runtime_common.c

```diff
@@ -1171,6 +1241,28 @@ wasm_runtime_deinstantiate_internal(WASMModuleInstanceCommon *module_inst,
 #endif
 }
 
+bool
+wasm_runtime_set_running_mode(wasm_module_inst_t module_inst,
+                              RunningMode running_mode)
+{
+#if WASM_ENABLE_FAST_JIT != 0 || WASM_ENABLE_JIT != 0
+    if (wasm_runtime_is_running_mode_supported(running_mode)) {
+        WASMModuleInstance *actual_module_inst =
+            (WASMModuleInstance *)module_inst;
+        actual_module_inst->e->running_mode = running_mode;
+        return true;
+    }
+#endif
+    return false;
+}
+
+RunningMode
+wasm_runtime_get_running_mode(wasm_module_inst_t module_inst)
+{
+    WASMModuleInstance *actual_module_inst = (WASMModuleInstance *)module_inst;
+    return actual_module_inst->e->running_mode;
```

Comment:
Should check whether module inst is AOT or Bytecode, if it is the former, return 0.